### ISSUE TITLE
Open Accessible View isn't working

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#26155]
Running the open accessible view command doesn't do anything for me. It doesn't work regardless of whether screen reader mode is on

Version: 1.80.0-insider (user setup)
Commit: 4649b5333370139d161e7b5d4f78ca13741efa35
Date: 2023-06-28T08:01:04.380Z
Electron: 22.3.14
ElectronBuildId: 21893604
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621
